hadoop-hdds/rocksdb-checkpoint-differ/src/main/java/org/apache/ozone/rocksdiff/RocksDiffUtils.java

```diff
@@ -113,4 +115,35 @@ public static boolean doesSstFileContainKeyRange(String filepath,
   }
 
 
+  @VisibleForTesting
```

Comment:
Remove this visibility tag.

Reply:
it is visible for testing since this shouldSkipNode function is being used only by tests and a function inside RocksDiffUtils. We could have made it private if not for the tests.